static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -366,6 +366,9 @@
 .email-subscription__button {
     @include fs-textSans(2);
     background: $guardian-brand;
+    width: $gs-baseline * 9;
+    height: $gs-gutter * 2;
+    border-radius: 62.5rem;
```

Comment:
I think we have a `px-to-rem` plugin in frontend so you can use pixels in sass files